osx/KA-Lite/KA-Lite/AppDelegate.h

```diff
@@ -30,8 +30,12 @@
 @property (strong, nonatomic) NSStatusItem *stopItem;
 @property (strong, nonatomic) NSStatusItem *openItem;
 @property (unsafe_unretained) IBOutlet NSTextView *taskLogs;
-@property signed int kaliteProcessCounter;
+
+// It will increment if port 8008 is not available.
 @property signed int portAlertCounter;
+// It will increment when startFunction() will execute.
+@property signed int kaliteStartCounter;
```

Comment:
Does this need to be an `int`?  How about using a `Boolean` instead as I saw below that not much increment/decrement/computation is happening on this variable.